templates/index.html

```diff
@@ -487,6 +487,12 @@ <h2 id="heading"></h2>
                                     <div class="col">
                                         <p id="right-sensor" class="text-center">Right Sensor: 0.000</p>
                                     </div>
+                                    <div class="col">
```

Comment:
Could this be changed to have the limit switches on a separate line from the line followers?